### DISCUSSION TITLE
ModuleNotFoundError: No module named 'gi'

### DISCUSSION THREAD

**Discussion by TorinKR** [Homebrew/discussions#148] [Everyday usage]
Hi! After upgrading to Big Sur, I ran brew update, then brew upgrade, and the vast majority of programs work. But virt-manager doesn't start with the message:
ModuleNotFoundError: No module named 'gi'
How to fix it????

Torin

**Comment by SMillerDev** [comment]
If you check brew doctor you'll find that we don't support Big Sur yet. In addition to that we don't really do code support for applications. Please report this to the virt-manager issuetracker.

> We do not provide support for this released but not yet supported version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Discourse, Twitter or any other official channels. You are responsible for
resolving any issues you experience while you are running this
released but not yet supported version.

**Comment by TorinKR** [comment]
Thank you for the information, I know that everything is not working at Big Sur yet, so I'm waiting patiently. I will also report virt-manager to the caregiver.
